crates/typst-library/src/foundations/str.rs

```diff
@@ -793,6 +817,18 @@ cast! {
     v: Str => Self::Str(v),
 }
 
+#[derive(PartialEq, Cast)]
+pub enum UnicodeNormalForm {
+    #[string("NFC")]
+    Nfc,
```

Comment:
the enum variants should have a bit of docs (at least one line) each, as that is what will be rendered in the docs. example: https://typst.app/docs/reference/text/highlight/#parameters-top-edge